ircd/send.c

```diff
@@ -1163,34 +1163,33 @@ sendto_local_clients_with_capability(int cap, const char *pattern, ...)
 	msgbuf_cache_free(&msgbuf_cache);
 }
 
-/* sendto_monitor()
+/*
+ * _sendto_monitor_with_capability_butserial()
  *
- * inputs	- monitor nick to send to, format, va_args
- * outputs	- message to local users monitoring the given nick
- * side effects -
+ * Shared implementation of sendto_monitor_with_capability_butserial and _sendto_monitor
  */
 void
-sendto_monitor(struct Client *source_p, struct monitor *monptr, const char *pattern, ...)
+_sendto_monitor_with_capability_butserial(struct Client *source_p, struct monitor *monptr, int caps, int negcaps, bool skipserial, const char *pattern, va_list * args)
```

Comment:
Eliminate current_serial abuse

```suggestion
_sendto_monitor_with_capability(struct Client *source_p, struct monitor *monptr, int caps, int negcaps, const char *pattern, va_list * args)
```